helm_deploy/values-prod.yaml

```diff
@@ -17,7 +17,10 @@ generic-service:
     WHEREABOUTS_API_URL: https://whereabouts-api.service.justice.gov.uk
     APPLICATIONINSIGHTS_CONFIGURATION_FILE: applicationinsights.json
 
-# CloudPlatform AlertManager receiver to route prometheus alerts to slack
-# See https://user-guide.cloud-platform.service.justice.gov.uk/documentation/monitoring-an-app/how-to-create-alarms.html#creating-your-own-custom-alerts
 generic-prometheus-alerts:
-  alertSeverity: digital-prison-service
+  alertSeverity: prison-visit-booking-alerts
+  sqsAlertsQueueNames:
+    - "Digital-Prison-Services-dev-hmpps_prison_visits_event_queue"
```

Comment:
Same as above, does the queue name need to change?

Reply:
this is what  other project have will double check